rts/lra/coordinator-quarkus/pom.xml

```diff
@@ -125,6 +130,35 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>native-docker</id>
+            <activation>
+                <property>
+                    <name>native-docker</name>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>io.quarkus</groupId>
+                        <artifactId>quarkus-maven-plugin</artifactId>
+                        <version>${version.quarkus}</version>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>native-image</goal>
+                                </goals>
+                                <configuration>
+                                    <enableHttpUrlHandler>true</enableHttpUrlHandler>
+                                    <dockerBuild>true</dockerBuild>
+                                    <containerRuntime>${quarkus.native.container-runtime}</containerRuntime>
```

Comment:
I will add the request for change directly here, please can you add a quickstart PR to use this image?

Reply:
@tomjenkinson Your comment implies that we are going ahead with this. I mentioned above about us not wanting to provide two native coordinators in two different projects. Quarkus is the place where deliver native support - and we have a native test for it already [1] and I have a quickstart somewhere.

Having two different places where users can obtain a native coordinator both dilutes our message and increases the support workload on the narayana team. I would prefer that the native is supported via the quarkus project where the expertise resides. Can you clarify your intention here.

[1] https://github.com/quarkusio/quarkus/pull/6763/files#diff-1c7e7b28df509e2d2fea89c3ba1bf65d3442637def31c479462fba48526adfa8R5

Reply:
I see, my intention would be if we were going ahead then we would need some way to verify the coordinator we are producing is functional. But if Narayana project isn't going ahead then I guess this PR and JBTM-3385 would be closed.